lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -910,6 +913,39 @@ export class UserAgentApplication {
     return adalFrame;
   }
 
+  private pollWindowForHash(contentWindow: Window): Promise<string> {
```

Comment:
This doesn't rely on anything in UAA, can it be in its own file ?

Reply:
Yeah, it would be good to put in an iframe utils file.